en/docs/how-tos/support-custom-claims-mapping.md

```diff
@@ -0,0 +1,99 @@
+# Support Custom Claims Mapping
+
+When JWT retrieved from Multiple Identity providers, Microgateway can map the relevant claims as a supported one to validates the JWT.  The claims of the incoming authentication JWT can differ based on the Authorization server that issued the JWT token. This feature maps the incoming jwt remote claims to the local claims of the microgateway as well as the claims expected by the back end service.  You can use custom claims mapping transformation in micro gateway when you have different keys or values of claims in your JWT token to make it as a Microgateway supported one.
```

Comment:
When JWT retrieved from Multiple Identity providers, Microgateway can map the relevant claims as a supported one to **validate** the JWT.  The claims of the incoming authentication JWT can differ based on the Authorization server that issued the JWT token. This feature maps the incoming **JWT** remote claims to the local claims of the **Microgateway** as well as the claims expected by the back end service.  You can use custom claims mapping transformation in micro gateway when you have different keys or values of claims in your JWT token to make it as a Microgateway supported one.